### PR TITLE
Changed `luakit://introspector/` to `luakit://binds/` (#446)

### PR DESCRIPTION
**Pull request by ghost**
A fix for #446 :)

### CODE REVIEW

**Comment by aidanholm** [comment]
Thanks for the PR! This looks good, but also needs a compatibility wrapper: i.e. `introspector_chrome.lua` should now print a warning and just `return require "binds_chrome"`

File: lib/binds_chrome.lua [hunk 1]
```diff
@@ -25,7 +25,7 @@ local html_template = [==[
 <html>
 <head>
     <meta charset="utf-8">
-    <title>Luakit Introspector</title>
+    <title>Luakit Key Bindings</title>
```
Comment (aidanholm) [inline lib/binds_chrome.lua]: I think this should just be "bindings", not "key bindings", as the user can bind mouse buttons and scroll events as well. Ditto everywhere else in this PR.

**Review by aidanholm — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Fixed the fix. ^^

**Comment by aidanholm** [comment]
Minor test failures:
```
FAIL style/test_source_format / test_vim_modeline
  tests/style/test_source_format.lua:41: Some files do not have modelines:
  lib/introspector_chrome.lua                            Missing/malformed modeline
FAIL style/test_source_format / test_lua_module_uses_M
  tests/style/test_source_format.lua:204: Some Lua modules have module table declaration errors:
  lib/introspector_chrome.lua                            Missing/malformed module table declaration
```
Please copy the `-- vim: ...` modeline into `introspector_chrome.lua`, and add an exception for that file in `tests/style/test_source_format.lua`, around line 177. Other than that, everything looks fine!

**Comment by ghost** [comment]
Thanks for the remarks!
By the way, how do I run the tests?

File: tests/style/test_source_format.lua [hunk 2]
```diff
@@ -175,9 +175,10 @@ end
 
 function T.test_lua_module_uses_M ()
     local exclude_files = {
-        "lib/markdown%.lua",   -- External file
-        "lib/.*/init%.lua$",   -- Module groupings
-        "lib/widget/%S*%.lua", -- Status bar widgets
+        "lib/markdown%.lua",        -- External file
+        "lib/.*/init%.lua$",        -- Module groupings
+        "lib/widget/%S*%.lua",      -- Status bar widgets
+        "lib/inspector_chrome.lua", -- Deprecated module
```
Comment (aidanholm) [inline tests/style/test_source_format.lua]: introspector, not inspector :)

**Review by aidanholm — CHANGES_REQUESTED** [review]

**Comment by aidanholm** [comment]
Tests are run with `make run-tests`; there's some extra dependencies you need though: luacheck, luassert, and xvfb

**Comment by ghost** [comment]
This time for sure...

**Comment by aidanholm** [comment]
Ok, great! Last two steps:

1) fixup all of the commits into 124bc8e6, _except_ for 24847ba4.
2) add a changelog entry

Let me know if you need help with the fixup step

**Comment by ghost** [comment]
Kept those two commits.

**Comment by aidanholm** [comment]
Thanks :)

**Pull request merged**